assessment_software/src/app/textarea-component/textarea.component.html

```diff
@@ -0,0 +1,7 @@
+<textarea rows="15" 
+cols="20"
+[ngClass]="styleClass"
+[maxLength]="maxlength"
+[minLength]="minlength"
+(input)="newTextAreaValue($event)"
+></textarea>
```

Comment:
When you try to edit, it shows a border on the text area. Was it intentional? 

![image](https://github.com/Aben-Bel/assessment_software/assets/44743368/f3ecf0d0-cefb-4892-8459-d1b389a43a09)

when it is not selected:

![image](https://github.com/Aben-Bel/assessment_software/assets/44743368/24e57020-e9e5-4e20-a780-e069faec860b)


It is better for the user to decide whether to add border on focus.